packages/remirror__extension-list/__tests__/list-keymap.spec.ts

```diff
@@ -0,0 +1,87 @@
+import { renderEditor, TaggedProsemirrorNode } from 'jest-remirror';
+import { BulletListExtension, OrderedListExtension, TaskListExtension } from 'remirror/extensions';
+
+describe('Enter', () => {
+  const editor = renderEditor([
+    new TaskListExtension({}),
+    new BulletListExtension({}),
+    new OrderedListExtension(),
+    new TaskListExtension(),
+  ]);
+  const {
+    nodes: { doc, paragraph: p, orderedList: ol, bulletList: ul, listItem: li, taskList: tl },
+    attributeNodes: { taskListItem },
+  } = editor;
+  const tif = taskListItem({ checked: false });
+  const tit = taskListItem({ checked: true });
```

Comment:
😆 Maybe a different name for this variable. It has a different meaning in English. I'm adding a suggestion, so you know what I'm referring to.

```suggestion
  const checked = taskListItem({ checked: true });
```

Reply:
😂 I didn't notice that. Sorry about this.

Reply:
Renamed:

- `tif` => `uncheck`
- `tit` => `checked`